src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java

```diff
@@ -94,7 +97,7 @@
  * </li>
  * <li>
  * Property {@code allowedAnnotations} - Specify annotations that allow
- * missed documentation. Only short names are allowed, e.g. {@code Generated}.
+ * tag violations. Only short names are allowed, e.g. {@code Generated}.
```

Comment:
that allows skip validation at all.

Reply:
yeah sure. I ran it locally and `allowedAnnotations` does skip validation for every possible violation in this check

```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="JavadocType">
            <property name="versionFormat" value="\$Revision.*\$"/>
            <property name="authorFormat" value="\S"/>
        </module>
    </module>
</module>
```

```
/**
 * @version invalid_version
 * @unknownTag value
 * @param K
 */
@Generated
public class ClassA<T> {
}
```

Command
```
java  -jar ../checkstyle-10.7.0-all.jar -c example_ten_checks.xml ./src/ExampleTen.java
```

Result
```
Starting audit...
Audit done.
```

Result without the @Generated annotation
```
Starting audit...
[ERROR] D:\RepoSense\try\.\src\ExampleTen.java:3:4: Unknown tag 'unknownTag'. [JavadocType]
[ERROR] D:\RepoSense\try\.\src\ExampleTen.java:4:4: Unused @param tag for 'K'. [JavadocType]
[ERROR] D:\RepoSense\try\.\src\ExampleTen.java:6:1: Type Javadoc comment is missing @author tag. [JavadocType]
[ERROR] D:\RepoSense\try\.\src\ExampleTen.java:6:1: Type Javadoc comment is missing @param <T> tag. [JavadocType]
[ERROR] D:\RepoSense\try\.\src\ExampleTen.java:6:1: Type Javadoc tag @version must match pattern '\$Revision.*\$'. [JavadocType]
Audit done.
Checkstyle ends with 5 errors.
```